### DISCUSSION TITLE
Disable collapsing/expanding tabs animation (via css?) while leaving animation enabled in settings?

### DISCUSSION THREAD

**Discussion by SXZ1** [piroor/treestyletab#3233] [Q&A] [answered]
Hello!
I am wondering if it's possible to disable collapsing/expanding tabs animation (preferably tab opening/closing animation as well) while leaving animation enabled in TST settings? I thought about using some css code but wiki only mentions other animations that can be disabled:

> @keyframes throbber {}
@keyframes tab-burst-animation {}
@keyframes tab-burst-animation-light {}
@keyframes blink {}

Why would I need that? I believe that's gonna speed up collapsing/expanding tab stacks with big number of tabs as the code that slows down collapsing/expanding gets executed only when animation is disabled in TST settings.

**Comment by irvinm** [comment, marked as answer]
Maybe try changing the values you want to disable:  https://github.com/piroor/treestyletab/blob/b6fd8f4461f10a6cbee30e0abe848371d5a8acab/webextensions/sidebar/styles/base.css#L29-L47

**Reply by SXZ1** [reply]
I've just tried this code and it does indeed affect collapsing/expanding animation yet doesn't bring any performance improvements, unfortunately. Thank you very much!

**Comment by piroor** [comment]
TST's animations are disabled when you disable animation effects with the setting of the platform itself. It looks better choice if battery usage seriously need to be reduced.
